### PR TITLE
lldpd: fix reload behaviour: advertisements shall default to on

### PR DESCRIPTION
**Pull request by systemcrash**
Because these capability advertisements default to on in lldpd, they became absent at reload, and not restart, due to how the reload logic works ( keep daemon running, send unconfigured and then the new config via socket ), and it was not evident unless you happened to be looking for it (e.g. via pcap or tcpdump). It was also not evident from the manpage ( have now sent patches upstream ).

At reload time, the unconfigure logic disabled them unless they were explicitly enabled (compare with other settings where 'unconfigure' just resets them). Now they default to on/enabled at init time, and are explicitly 'unconfigure'd at startup if the user disables them via:

lldp_mgmt_addr_advertisements=0
lldp_capability_advertisements=0

In other words: explicit is necessary to disable the advertisements.

The same applies to 'configure system capabilities enabled'. Technically 'unconfigure'd is the default but now it is explicit at reload.

Tested on: 23.05.3

FAO: @Ansuel @robimarko @stintel @pprindeville

### CODE REVIEW

**Comment by robimarko** [comment]
Thanks! Rebased on top of main and merged!

**Comment by systemcrash** [comment]
Thanks @robimarko!
